docs/_sidebar.md

```diff
@@ -2,26 +2,51 @@
 landing page of this section. Other subsections are placed below and can be linked separately. Feel
 free to add sections and subsections to this sidebar.)
 
-[comment]: <> (Html instead of markdown due to known issues: https://github.com/docsifyjs/docsify/issues/850
-and https://github.com/docsifyjs/docsify/issues/1139)
-
-- <a href="#/README">Home</a>
-
-- Getting Started
-  - <a href="#/submodule/GitHub/KNOWN_FRIENDS.md">Adoptions</a>
-  - <a href="#/submodule/GitHub/CONTRIBUTING.md">Contributing</a>
-  - <a href="#/hands-on.md">Hands-on</a>
-
-- Components
-  - <a href="#/submodule/Connector/">Connector</a>
-  - <a href="#/submodule/DataDashboard/">Data Dashboard</a>
-  - <a href="#/submodule/FederatedCatalog/">Federated Catalog</a>
-  - <a href="#/submodule/GradlePlugins/">Gradle Plugins</a>
-  - <a href="#/submodule/IdentityHub/">Identity Hub</a>
-  - <a href="#/submodule/MinimumViableDataspace/">Minimum Viable Dataspace</a>
-  - <a href="#/submodule/RegistrationService/">Registration Service</a>
-  - <a href="#/submodule/TrustFrameworkAdoption/">Trust Framework Adoption</a>
-
-- Documents
-  - <a href="#/submodule/Collateral/">Collateral</a>
-  - <a href="#publications.md">Publications</a>
\ No newline at end of file
+- [Home](/README)
+
+&nbsp;&nbsp;**Getting Started**
+- [Adoptions](/submodule/GitHub/KNOWN_FRIENDS.md)
+- [Contributing](/submodule/GitHub/CONTRIBUTING.md)
+- [Hands-on](/hands-on.md)
+
+&nbsp;&nbsp;**Components**
+
+- [Connector](/submodule/Connector/)
+  - [Developer](/submodule/Connector/docs/developer/)
```

Comment:
```suggestion
  - [Documentation](/submodule/Connector/docs/developer/)
```
Would call all of them `Documentation` (or `Developer Documentation`).

Reply:
Better `Documentation` because we will have `Developer Documentation` in the getting started section.